recipes/cartographer/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "cartographer" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/cartographer-project/cartographer/archive/{{ version }}.tar.gz
+    sha256: 474a410bf6457eb8a0fd92ea412d7889fb013051e625d3ee25e8d65e4113fd6c
+    patches:
+      - fix-windows.patch  # [win]
+
+build:
+  number: 0
+  skip: true  # [unix or win and vc<14]
```

Should be enough to skip unix for now, we don't build for `vc<14` anymore.
```suggestion
  skip: true  # [unix]
```